src/controllers/MigrationController.php

```diff
@@ -433,6 +433,27 @@ public function actionCreate($table)
     }
 
     /**
+     * Creates new migrations for every table in your database.
+     * @throws InvalidParamException
+     */
+    public function actionCreateAll()
+    {
+        $tables = $this->db->schema->getTableNames();
+
+        if (!$tables) {
+            return $this->stdout("Your Database does not contain any tables yet.");
+        }
+
+        if ($this->prompt("Are you sure you want to generate " . count($tables) . " migrations? [yes/no]") === 'yes') {
+            foreach ($tables as $table) {
+                $this->actionCreate($table);
```

Comment:
Instead of iterating over the tables you can call this action once and pass the imploded list of names - thanks to this the same mass decision can be used for each table if user would like to change the default one in case of file to be generated already exists.